### ISSUE TITLE
URL special character encoding inconsistent

### ISSUE THREAD

**Issue by georgecrawford** [ben-eb/postcss-normalize-url#14]
```
$ cat file.scss
.class {
  background-image: url("https://example.com?tint=%23505050,%23505050");
}

$ cssnano file.scss
.class{background-image:url(https://example.com/?tint=%23505050,#505050)}

$ cssnano file.scss --no-normalizeUrl
.class{background-image:url("https://example.com?tint=%23505050,%23505050”)}
```

The encoding of `#` as `%23` is desired here, and the un-encoding behaviour is certainly inconsistent (only the first instance is converted). I'm not sure what `normalizeUrl` is trying to do here.

**Comment by georgecrawford** [comment]
I've reproduced this in https://github.com/sindresorhus/normalize-url/issues/26.

**Comment by ben-eb** [comment]
Fixed in Node `v4.7.0`.

https://github.com/sindresorhus/normalize-url/issues/26#issuecomment-266317693
